packages/front-end/components/DataViz/SqlExplorerDataVisualization.tsx

```diff
@@ -128,21 +130,24 @@ export function DataVisualizationDisplay({
   dataVizConfig: Partial<DataVizConfig>;
 }) {
   const isConfigValid = useMemo(() => {
-    const parsed = dataVizConfigValidator.strip().safeParse(dataVizConfig);
+    const parsed = dataVizConfigValidator.safeParse(dataVizConfig);
```

Comment:
question: Why are we removing the `strip()`?

Reply:
We're now using the `discriminatedUnion` which doesn't play well with `.strip()` and AFAIK, discriminatedUnion has it's own built-in validation.

Reply:
My concern is that we might fail validation when moving from a config that has dimension to one that doesn't and we don't completely reset the object.

When testing this worked as expected though, so I think my concern might be unfounded.